src/Orchard/Recipes/Services/IRecipeBuilderStepResolver.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
```

Comment:
Added RecipeBuilderStepResolver and RecipeExecutionStepResolver for convenience.